docs/devel/releasing.md

```diff
@@ -265,6 +266,12 @@ can, for instance, tell it to override `gitVersion` and set it to
 `v0.4-13-g4567bcdef6789-dirty` and set `gitCommit` to `4567bcdef6789...` which
 is the complete SHA1 of the (dirty) tree used at build time.
 
+## Updating the master branch
+
+If you are cutting a new release series, please also update the master branch:
+change the `latestReleaseBranch` in `cmd/mungedocs/mungedocs.go` to the new
+release, and run `hack/update-generated-docs.sh`. This will let the unversioned
+warning in docs point to the latest release series.
```

Comment:
Presumably you also want this person to send a PR?  Can you add something explicit about that, what to call it, and whom to assign it to?